bin/taxref_reformat_qiime_greengenes2022.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+# Decompress files.
+gzip -c -d 2022.10.seqs.fna.gz > 2022.10.seqs.fna
+gzip -c -d 2022.10.taxonomy.md5.tsv.gz > 2022.10.taxonomy.md5.tsv
+
+# Select and rename files
+mv *.fna greengenes2022.fna
+mv *.tsv greengenes2022.tax
```

Comment:
Why not a little more dynamic to make it work also with other versions, for example
```suggestion
gzip -c -d *.seqs.fna.gz > greengenes2.fna
gzip -c -d *.taxonomy.md5.tsv.gz > greengenes2.tax
```